UniversalAlgebra.tex

```diff
@@ -67,30 +67,31 @@ \section{Universe, Type, and Signature}
 
 \begin{itemize}
 
-    \item A \textit{signature} is a pair $\Sigma = (S,F)$ such that $S$ is a set
-    of single sort. $F$ is the collection of operation names.
-
-    \item A \emph{$\Sigma$-algebra} $A$ is defined as pair $A = (A,F_A)$, a
-    mathematical structure that has a domain $(A)$ and a family of functions
-    $(F_A)$ defined for each function symbol in the signature on this domain.
-    Algebra provides an interpretation for the carrier set $A$ and function
-    symbols $F_A$ of a theory.
-
-    \item Let $E$ be a set of equations, then model $(Mod_\Sigma(E))$ denotes
-    the class of $\Sigma$-algebra satisfying all the equations in $E$.
-    \[Mod_\Sigma(E) = \{ A | A \text{ is a } \Sigma-algebra \text{ and A } \models E \} =
-    Mod[⟨\Sigma,E⟩]\]
+    \item A \textit{signature} is a pair $\Sigma = (S,F)$ such that $S$ is a
+    carrier set. $F$ is the collection of operation names.
+
+    \item A \emph{$\Sigma$-algebra} in \cite{sannella2012foundations} is defined
+    as: "A $\Sigma$-algebra $A$ consists of an S-Sorted set $|A|$ of carrier
```

Comment:
The issue is that you are doing single-sorted universal algebra and Sanella does multi-sorted. You should find a textbook that does single-sorted, it will simplify your life.

Reply:
Thank you. I changed definition from "Wolfgang Wechler's - Universal Algebra for Computer Scientists"